operators/pkg/controller/remotecluster/remotecluster_controller_integration_test.go

```diff
@@ -94,12 +94,10 @@ func TestReconcile(t *testing.T) {
 		if err != nil {
 			return err
 		}
-		switch {
-		case len(trustRelationship1.Spec.CaCert) == 0:
-			return errors.New("Not reconciled yet")
-		default:
-			return nil
+		if len(trustRelationship2.Spec.CaCert) == 0 {
+			return errors.New("no ca cert reconciled yet")
 		}
+		return nil
```

Comment:
Well seen! Maybe we can also replace the `switch` by an `if` here: https://github.com/elastic/cloud-on-k8s/blob/2b56d276353b0d951a67a2c2ef26f6f44330757b/operators/pkg/controller/remotecluster/remotecluster_controller_integration_test.go#L83-L87

Reply:
That's fixed in another commit of that PR (sorry for not making it consistent in the same commit).